opentelemetry-sdk/tests/trace/export/test_export.py

```diff
@@ -480,11 +484,6 @@ def test_batch_span_processor_reset_timeout(self):
 
         span_processor.shutdown()
 
-    if python_implementation() == "PyPy":
-        test_batch_span_processor_reset_timeout = flaky(
-            max_runs=2, min_passes=1
```

Comment:
The test ignores the "max_runs" property of flaky?

Reply:
It doesn´t, it just happens to fail sometimes two times in a row and ends up failing CI.